test/controllers/diary_entry_controller_test.rb

```diff
@@ -641,6 +662,72 @@ def test_comments
     assert_response :not_found
   end
 
+  ##
+  # test subscribe success
+  def test_subscribe_success
```

Comment:
I'm not sure the comments here are adding anything - they are just repeating the name of the test without the underscores. (+further 3 times below)